gnd/src/main/java/com/google/android/gnd/ui/map/ExtentSelector.java

```diff
@@ -0,0 +1,28 @@
+package com.google.android.gnd.ui.map;
+
+import android.util.Pair;
+
+import androidx.annotation.Nullable;
+
+import com.google.android.gnd.model.feature.Feature;
+import com.google.common.collect.ImmutableSet;
+
+import io.reactivex.Observable;
+
+public interface ExtentSelector extends MapProvider.MapAdapter {
+
+  Observable<Extent> getExtentSelections();
+
+  void updateExtentSelections(ImmutableSet<Extent> extents);
```

Comment:
Do extents need to be updated in batch this way, or is it possible to simplify this API? (e.g. `updateExtent(extent)`?

Reply:
It probably is possible to make it simpler--it was easier to start off this way since we fetch tiles as a set from the DB;--I've added a TODO to revisit this.